### ISSUE TITLE
Cannot deactivate "no-console" rule.

### ISSUE THREAD

**Issue by Moezalez** [eslint/eslint#4100]
I am using node and want to use console.log. I cannot figure out a way to prevent eslint to warn me about it. I get `Unexpected console statement.`

My eslintrc:
```
{
    "rules": {
        "indent": [
            2,
            2
        ],
        "quotes": [
            2,
            "single"
        ],
        "linebreak-style": [
            2,
            "unix"
        ],
        "semi": [
            2,
            "always"
        ]
    },
    "env": {
        "es6": true,
        "node": true
    },
    "extends": "eslint:recommended"
}
```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by gyandeeps** [comment]
Can you share code where the issue is?
What version of eslint are you using?

Insode your rules just say `no-console: 0`

**Comment by ilyavolodin** [comment]
@Moezalez your config file includes `"extends": "eslint:recommended"` which means that you will inherit all of the ESLint default rules, including `no-console`. In order to disable `no-console` you should include the following into your `rules` section:
`no-console: 0`

Let us know if you have any trouble getting that to work.

**Comment by Moezalez** [comment]
Code:

```javascript
var express = require('express');
var app = express();

app.get('/', function (req, res) {
  res.send('Hello World!');
});

var server = app.listen(3000, function () {
  var host = server.address().address;
  var port = server.address().port;

  console.log('Example app listening at http://%s:%s', host, port);
});
```

config:
```{
    "rules": {
        "indent": [
            2,
            2
        ],
        "quotes": [
            2,
            "single"
        ],
        "linebreak-style": [
            2,
            "unix"
        ],
        "semi": [
            2,
            "always"
        ]
    },
    "env": {
        "es6": true,
        "node": true
    },
    "no-console": 0,
    "extends": "eslint:recommended"
}
```

I still get 
```
  12:3  error  Unexpected console statement  no-console

✖ 1 problem (1 error, 0 warnings)
```

**Comment by michaelficarra** [comment]
@moezalez You didn't put it in the rules section as asked.

**Comment by ilyavolodin** [comment]
@Moezalez as @michaelficarra noted, `no-console` should go inside `rules` section.

**Comment by smarzougui** [comment]
Hi guys,
I've inserted it in the rules section as below:

`{
    "env": {
        "browser": true,
        "commonjs": true,
        "es6": true
    },
    "extends": "eslint:recommended",
    "parserOptions": {
        "sourceType": "module"
    },
    "rules": {
        "no-console": 0,
        "indent": [
            "error",
            "tab"
        ],
        "linebreak-style": [
            "error",
            "unix"
        ],
        "quotes": [
            "error",
            "double"
        ],
        "semi": [
            "error",
            "never"
        ]
    }
}
`

but I keep seeing the   Unexpected console statement  no-console

What can be the issue ?

**Comment by platinumazure** [comment]
@smarzougui Try running ESLint with the `--print-config` option for the file in question. It will show you the full configuration (with all extensions, parent configs, etc. merged in) for that file. You can also run with the `--debug` option to see what config files are being pulled in.

If you have further difficulties, please stop by our [Gitter chat](https://gitter.im/eslint/eslint) or open a new issue. Thanks!
